controller-server/src/main/java/com/yahoo/vespa/hosted/controller/ApplicationController.java

```diff
@@ -539,17 +539,8 @@ public void restart(DeploymentId deploymentId, Optional<Hostname> hostname) {
 
     /** Deactivate application in the given zone */
     public void deactivate(Application application, ZoneId zone) {
-        deactivate(application, zone, Optional.empty(), false);
-    }
-
-    /** Deactivate a known deployment of the given application */
-    public void deactivate(Application application, Deployment deployment, boolean requireThatDeploymentHasExpired) {
-        deactivate(application, deployment.zone(), Optional.of(deployment), requireThatDeploymentHasExpired);
-    }
-
-    private void deactivate(Application application, ZoneId zone, Optional<Deployment> deployment,
-                            boolean requireThatDeploymentHasExpired) {
-        if (requireThatDeploymentHasExpired && deployment.isPresent()
+        Optional<Deployment> deployment = Optional.empty();
+        if (false && deployment.isPresent()
```

Comment:
`if (false`?

Reply:
Huh? I clearly remember deleting those lines of code >_<